2001-3000/LIT2892RepCh173.xml

```diff
@@ -58,13 +59,19 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
                <bibl>
                   <ptr target="bm:Chaine1913Rep"/>
                   <citedRange unit="item">173</citedRange>
+                  <note>Chaine mistakenly attributed this malkǝʾ-hymn to Walatta Ṗeṭros in his Repertoire des Salam, but correctly attributed the hymn to ʾƎḫǝta Krǝstos in <bibl>
```

Comment:
Mark up Repertoire des Salam with bm:Chaine1913Rep (or bm:Chaine1913RepPart1 or bm:Chaine1913RepPart2, if you know precisely).